src/hooks/useVibeMediaQuery/__stories__/useVibeMediaQuery.stories.mdx

```diff
@@ -0,0 +1,39 @@
+import { Story, Canvas, Meta } from "@storybook/addon-docs";
+import useVibeMediaQuery from "..";
+import "../../__stories__/general-hooks-stories.scss";
+import styles from "./useVibeMediaQuery.modules.scss";
+
+
+<Meta title="Hooks/useVibeMediaQuery" />
+
+# useVibeMediaQuery
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Feedback](#feedback)
+
+## Overview
+
+This hook will return the value of the current vibe break point
+
+| Value | Media query |
+|-------|-------------|
+| 1     | screen and (max-width: 767px)            |
```

Comment:
Why not writing here the names of the enums the same as in the code?

Reply:
this is the value which returns from the function